static/js/modules/typeahead.js

```diff
@@ -0,0 +1,152 @@
+'use strict';
+var Handlebars = require('handlebars');
+var events = require('./events.js');
+
+module.exports = {
+  init: function() {
+    function filterCandidates(result) {
+      // Label House, President, Senate
+      var officeFull;
+      switch(result.elections[0].office_sought) {
```

Comment:
It would be a little bit nicer if we could map these values instead of using a `switch` statement. Something like:

```
var officeMap = {
  'H': 'House of Representatives'
  // and so on
}

officeFull = officeMap[result.elections[0].office_sought]
```